src/Cake.Common/Tools/OpenCover/OpenCoverSettings.cs

```diff
@@ -115,6 +120,7 @@ public OpenCoverSettings()
             _excludedFileFilters = new HashSet<string>(StringComparer.Ordinal);
             Register = "user";
             LogLevel = OpenCoverLogLevel.Info;
+            HideSkippedOption = OpenCoverHideSkippedOption.All;
```

Comment:
Again, I think the default is empty/none?

Reply:
Defaulted it to `None`